lib/resource-mapper.js

```diff
@@ -69,7 +69,7 @@ class ResourceMapper {
   async mapFileToUrl ({ path, hostname }) {
     // Remove the root path if specified
     path = path.replace(/\\/g, '/')
-    if (path.startsWith(this._rootPath)) {
+	if (path.startsWith(this._rootPath)) {
```

Comment:
please undo this indent, it was correct with the `i` of if under the `p` of path

Reply:
Done.

Reply:
You added a ";". Lint impose a very strict style. There should be no ;

Reply:
OK, I didn't realize. I've removed the ;